tldextract/remote.py

```diff
@@ -38,7 +38,9 @@ def find_first_response(urls):
         except requests.exceptions.RequestException as ree:
             LOG.error('Exception reading Public Suffix List url ' + url + ' - ' + str(ree) + '.')
         else:
```

Comment:
This doesn't close `session` if it was opened but there's an exception. I'll switch to using it as a context manager. This should still be readable, without crazy `try` nesting.